src/style.css

```diff
@@ -1 +1,13 @@
-/* styles go here */
+.form__fieldset {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.form__fieldset > .form__lable:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+.radio .form__input {
+  margin-left: 10px;
+}
```

Comment:
do you need these styles? 
When i opened your test link there were no applied styles for these inputs

Reply:
It is surprising, than the tests did not pass without it, but now everything is ok.